src/value_generators/test/DocumentGeneratorTestCases.yml

```diff
@@ -498,4 +498,40 @@ Tests:
     #    - { "int" : { "$numberLong" : "2000000000"}, "string" : "2000000000" }
     - "string": "2"
 
+  # Now moves with time so would always fail.
+  # - Name: Now
+  #   GivenTemplate:
+  #     date: {^Now: {}}
+  #   ThenReturns:
+  #   - { "date" : { "$date" : { "$numberLong" : "1453689676330" } } }
 
+  - Name: DateMinMax
+    GivenTemplate:
+      date: {^Date: {min: "2020-01-01", max: "2021-01-01"}}
+    ThenReturns:
+    - { "date" : { "$date" : { "$numberLong" : "1606971090942" } } }
+
+  # The tests with no max generate different values every time as max moves.
```

Comment:
Can we make default max something constant?

Reply:
yes! max and min now have defaults.